docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,297 @@ have not been included. These are:
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
+error per selected string, but obviously will not be exact. To implement this, select the strings 
+eligible for noise and then apply the OCR noise function to all strings. 
+
+.. todo::
+
+  Include character based noise level (applies to phonetic and typographic too)
+
+Limitations: 
+
+- The result of having parameters at the character level is that we cannot tell exactly how many data points have noise in them. With the current settings, it is likely similar to the user selected value, but we will always have fewer errors than expected. This could be improved at a later time. 
+
+**Phonetic**
+
+Phonetic errors are when a character is misheard. This could similar sounding letters when 
+spoken like 't' and 'd' for example; or letters that make the same sounds within a word like 'o' and 'ou'. 
+In order to emulate that, there is a `GeCo like corrupter and related list of possible changes in the phonetic_variations csv found here <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
+error per selected string, but obviously will not be exact. To implement this, select the strings 
+eligible for noise and then apply the OCR noise function to these strings.
+
+Limitations: 
+
+- The result of having parameters at the character level is that we cannot tell exactly how many data points have noise in them. With the current settings, it is likely similar to the user selected value, but we will always have fewer errors than expected. This could be improved at a later time. 
+- Certain observers might be more likely to include phonetic errors, such as the census where large amounts of data are gathered over the phone or in person. However, we do not include that level of specificity in the design of the noise function. The amount of noise can always be changed at the observer level when extracting data to account for this. 
+
+
+**Typographic** 
+
+Typographic errors occur due to mistyping information. The commonality of errors are therefore 
+based on the QWERTY keyboard layout. These errors can include added characters, missed characters, 
+and replaced characters. In order to emulate that, there is a `GeCo like corrupter and related layout of the QWERTY keyboard csv found on github <https://github.com/ihmeuw/vivarium_research_prl/tree/main/noise>`_. 
+
+Currently, the user defined rate will be a rate 'eligible' for noise. The actual rate of noise 
+will be determined at the character level as XX errors per character. We think this approximates to 1 
```

Comment:
The [typographic noise function](https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/corruption.py#L119) actually has two user-specified probabilities instead of one: (1) the probability that a token is corrupted, and (2) the probability that a corrupted token contains the original token (a single character in this case) in addition to the corruption.